Resources/Private/JavaScript/asset-upload/src/components/FilePreview.tsx

```diff
@@ -1,35 +1,218 @@
+
+        setFiles((prev) => {
+            return { ...prev, selected: files };
+        });
+
+        setUploadPossible(getUploadPossibleValue(files));
+    };
+
+    const setCaption = (caption: string) => {
+        const files = setUploadProperty('caption', caption);
+
+        setFiles((prev) => {
+            return { ...prev, selected: files };
+        });
+
+        setUploadPossible(getUploadPossibleValue(files));
+    };
+
+    const setCopyrightNoticeNotNeeded = (isChecked: boolean) => {
+        setCopyrightNoticeNotNeededChecked(isChecked);
+        const files = setUploadProperty('copyrightNoticeNotNeeded', isChecked);
+
+        setFiles((prev) => {
+            return { ...prev, selected: files };
+        });
+
+        setUploadPossible(getUploadPossibleValue(files));
+    };
 
     // TODO: Output helpful localised messages for results 'EXISTS', 'ADDED', 'ERROR'
     return (
-        <div
-            className={cx(
-                classes.thumb,
-                error ? classes.error : success ? classes.success : loading && classes.loading
-            )}
-            title={file.name}
-        >
-            <div className={classes.thumbInner}>
-                <img src={file.preview} alt={file.name} className={classes.img} />
-                {loading && <Icon icon="spinner" spin={true} />}
-                {success && <Icon icon="check" />}
-                {error && <Icon icon="exclamation-circle" />}
-                {fileState?.result && <span>{fileState.result}</span>}
+        <div className={classes.preview}>
+            <div
+                className={cx(
+                    classes.thumb,
+                    error ? classes.error : success ? classes.success : loading && classes.loading
+                )}
+                title={file.name}
+            >
+                <div className={classes.thumbInner}>
+                    <img src={file.preview} alt={file.name} className={classes.img} />
+                    {loading && <Icon icon="spinner" spin={true} />}
+                    {success && <Icon icon="check" />}
+                    {error && <Icon icon="exclamation-circle" />}
+                    {result && <span>{result}</span>}
+                </div>
+            </div>
+            <div className={classes.properties}>
+                {uploadPropertiesConfig['title'].show ? (
+                    <Property label={translate('inspector.title', 'Title')}>
+                        <TextInput
+                            className={classes.textInput}
+                            disabled={disabled}
+                            value={file.title || ''}
+                            onChange={setTitle}
+                        />
+                    </Property>
+                ) : (
+                    ''
+                )}
+                {uploadPropertiesConfig['caption'].show ? (
+                    <Property label={translate('inspector.caption', 'Caption')}>
+                        <TextArea
```

Comment:
The textareas are missing the top and bottom padding due to a conflict with the module styles. So we need a stronger selector or important to style them.

Reply:
i added an `!important` to override the Neos module styles.